src/types/StatusFilter.ts

```diff
@@ -0,0 +1,5 @@
+export enum StatusFilter {
+  all = 'all',
```

Comment:
This is not a mistake, but it is customary to write enam names in capital letters ( `ALL = 'all'` ).